README.md

```diff
@@ -1,10 +1,60 @@
+
+This is the quickest, easiest and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues depending on where it was found:
+
+- For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose)
+- For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose)
+- For problems you've found in any other section of [docs](https://docs.quantum-computing.ibm.com), open a content bug issue [here](https://github.com/Qiskit/documentation/issues/new/choose).
+
+### 2. Suggest new content
+
+If you think there are gaps in our documentation, or sections that could be expanded upon, we invite you to open a new content request issue [here](https://github.com/Qiskit/documentation/issues/new/choose). 
+
+Not every new content suggestion is a good fit for docs, nor are we able to prioritise every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreicate our community's efforts in generating new ideas.
+
+If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted we will let you know and work with you to get the content written and reviewed.
+
+Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved and it is unlikely that PRs submitted without an associated approved content request will be accepted.
+
+### 3. Validate exisiting issues
+
+You can help the maintainer team prioritise already open issues by doing the following:
+
+- For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.  browser type, qiskit version etc.)
+- For new content requests, leave a comment or upvote (👍) in the issue if you also would like to see that new content added.
+
+### 4. Fix an open issue
+
+You can look through the open issues we have in this repo and address them with a PR. We recommend focusing on issues with the "good first issue" or "help wanted" label. 
+
+Before getting started on an issue remember to do the following:
+
+1. Read the Code of Conduct (TODO add link)
```

Comment:
Should we link to https://github.com/Qiskit/qiskit/blob/main/CODE_OF_CONDUCT.md for now?